build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "2.7.8"
+    id("io.spring.dependency-management") version "1.0.14.RELEASE"
+    java
```

Comment:
is this still needed?

Reply:
We also have kotlin("plugin.spring") version kotlinVersion no idea if this is necessary xD